x-pack/test/api_integration/apis/management/rollup/rollup.test_helpers.js

```diff
@@ -101,12 +101,27 @@ export const registerHelpers = ({ supertest, es }) => {
 
   const deleteIndicesGeneratedByJobs = () => (
     supertest.get(`${API_BASE_PATH}/indices`)
-      .then(({ body }) => {
+      .then(({ status, body }) => {
+
+        if (status !== 200) {
+          // It seems that there is a flakiness when requesting rollup indices
+          // If we did not succeed fetching the indices we won't be able to delete them.
+          // https://github.com/elastic/kibana/issues/33282
+          console.log(`[WARNING] Error fetching rollup indices with error: "${body}"`);
+          return;
+        }
+
         const index = Object.keys(body);
```

Comment:
@spalger this is probably where the `no such index [statusCode]` came from. It seems that in some cases the fetch fails and the body is the error instead of the indices.
Hopefully with the console log above we will understand better in which scenarios the

```js
supertest.get(`${API_BASE_PATH}/indices\`)
```

fails...